src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <section class="calendar">
+      <div class="calendar__field calendar__field--day-start--sun calendar__field--day-count--29">
```

Comment:
I suppose you can easily get rid of this div

Reply:
I agree with you

Reply:
I think it is made according to BEM, if necessary, I can easily move the section, because it is not tied to "page"